vocabularies/Common.xml

```diff
@@ -834,6 +839,11 @@ The minimum structure is
           </Collection>
         </Annotation>
       </ComplexType>
+      <ComplexType Name="ValueListRangeInOut" BaseType="Common.ValueListParameterInOut">
+        <Property Name="Operator" Type="UI.ValueListRangeOperator" DefaultValue="EQ" Nullable="false">
```

Comment:
```suggestion
        <Property Name="Operator" Type="UI.ValueListRangeOperator" DefaultValue="UI.ValueListRangeOperator/EQ" Nullable="false">
```

Reply:
Thx for the hint! 395a11a54059513ee6d8bed10c5cabf68de56832